packages/AddressSearch/utils/getAddresses.js

```diff
@@ -25,7 +23,7 @@ export const getAddressSuggestions = async (provider, query) => {
     ))
     return addresses
   } else if (provider === 'Mapbox') {
-    const response = await fetchData(MAPBOX_GEOCODER_API + query + MAPBOX_GEOCODER_PARAMS)
+    const response = await fetchData(MAPBOX_GEOCODER_API + encodeURIComponent(query) + mapboxGeocoderParams)
```

Comment:
This URI encode will get some unrelated results. Not sure if it's bug in the API or we shouldn't encode the search query.
![Screen Shot 2020-08-02 at 4 38 14 pm](https://user-images.githubusercontent.com/31641325/89117132-c15ac100-d4de-11ea-8e6f-b30f00b72ba5.png)